applications/hotornot/src/hon_util.erl

```diff
@@ -119,7 +119,7 @@ sort_rate(RateA, RateB) ->
 
     case PrefixA =:= PrefixB of
         'true' ->
-            wh_json:get_integer_value(<<"weight">>, RateA, 100) >
+            wh_json:get_integer_value(<<"weight">>, RateA, 100) <
                 wh_json:get_integer_value(<<"weight">>, RateB, 100);
         'false' ->
             PrefixA > PrefixB
```

Comment:
What about this one?

[Here](https://github.com/kalda341/kazoo/blob/patch-6/applications/hotornot/src/hon_util.erl#L114) it speaks of the weight, but how should the prefix be handled?
Can you give us a complete little table of in/outs so that we can reflect together on what code to write?